Jenkinsfile

```diff
@@ -1,9 +1,11 @@
+/*
+ See the documentation for more options:
+ https://github.com/jenkins-infra/pipeline-library/
+*/
 buildPlugin(
-  useContainerAgent: true,
+  forkCount: '1C', // run this number of tests in parallel for faster feedback.  If the number terminates with a 'C', the value will be multiplied by the number of available CPU cores
+  useContainerAgent: true, // Set to `false` if you need to use Docker for containerized tests
   configurations: [
-    [platform: 'linux', jdk: 11],
-    [platform: 'windows', jdk: 11],
-    [platform: 'linux', jdk: 17],
+    [platform: 'linux', jdk: 21],
```

Comment:
What is the reasoning behind testing different jdk version on `linux` vs `windows`?